internal/daemon/vendor-specific-plugins/common/vspnetutils.go

```diff
@@ -110,3 +132,134 @@ func GetNetDevNameFromPCIeAddr(platform platform.Platform, pcieAddress string) (
 
 	return "", fmt.Errorf("network device not found for PCI address %s", pcieAddress)
 }
+
+// CreateVethPair function to create a veth pair with the given index and InterfaceInfo
+func CreateVethPair(index int) (*VEthPairDeviceInfo, error) {
+	//nfInterfaceName is the name of the interface on the Network Function attached to the container
+	//dpInterfaceName is the name of the interface on the Data Plane OvS side
+	nfInterfaceName := fmt.Sprintf("nf_if%d", index)
+	dpInterfaceName := fmt.Sprintf("dp_if%d", index)
```

Comment:
I think there should be a function `CreateVethPair()` that takes the interface name and the peer name.

Choosing a particular name (nf_if%d) should be done by the caller, or a wrapper function `CreateNfVethPair()`.

Reply:
Good point. Changing this.